gpustack/cmd/start.py

```diff
@@ -363,6 +363,13 @@ def setup_start_cmd(subparsers: argparse._SubParsersAction):
         action='append',
         help='HTTP request headers allowed in cross-origin requests. Specify the flag multiple times for multiple headers. Example: --allow-headers Authorization --allow-headers Content-Type. Default: ["Authorization", "Content-Type"].',
     )
+    # authentication configuration
+    group.add_argument(
+        "--EXTERNAL_AUTH",
+        type=str,
+        help="A SSO authentication info.",
+        default=get_gpustack_env("authentication_info")
+    )
```

Comment:
It may be cumbersome to construct the auth config in a single env var. How about we use one argument for each attribute similar to https://docs.openwebui.com/features/sso/#oidc? And we can parse them to a structure defined in GPUStack config.